src/js/profile/subcategory.js

```diff
@@ -167,13 +175,12 @@ export class Subcategory extends Component {
                     $(wrapper).append(indicatorContainer);
                     let metadata = indicator.metadata;
                     if (indicator.content_type === ContentBlock.BLOCK_TYPES.Indicator) {
-                        block = this.addIndicatorBlock(indicatorContainer, indicator, title, isLast, addLockButton, restrictValues, defaultFilters);
+                        block = this.addIndicatorBlock(indicatorContainer, indicator, title, isLast, addLockButton, restrictValues, defaultFilters, hiddenIndicators);
                     } else if (indicator.content_type === ContentBlock.BLOCK_TYPES.HTMLBlock) {
-                        block = this.addHTMLBlock(indicatorContainer, indicator, title, isLast);
+                        block = this.addHTMLBlock(indicatorContainer, indicator, title, isLast, true, {}, [], hiddenIndicators);
```

Comment:
It looks like addHTMLBlock only expects four parameters

Reply:
Thanks @michaelglenister I have updated code  👍